src/mono/browser/runtime/debug.ts

```diff
@@ -66,7 +66,7 @@ function mono_wasm_malloc_and_set_debug_buffer (command_parameters: string) {
         if (_debugger_buffer)
             Module._free(_debugger_buffer);
```

Comment:
it might be good to introduce a 'free' in memory that's paired with our new malloc, even if it just calls _free